src/site/layouts/basic_landing_page.drupal.liquid

```diff
@@ -50,6 +50,58 @@
             {% include "src/site/includes/support_resources_search_bar.drupal.liquid" %}
           </div>
 
+          <div class="usa-alert usa-alert-info" role="alert">
+            <div class="usa-alert-body">
+                <h3 class="usa-alert-heading">
+                    We're currently in beta testing
+                </h3>
+                <div class="processed-content">
+                  <div itemprop="text">
+                    <p>Welcome to resources and support, a new part of <a href="/">VA.gov</a>. We'll be adding more articles and topics in the weeks and months ahead, so please check back often.</p>
+                  </div>
+                </div>
+            </div>
+          </div>
```

Comment:
This "beta" banner is part of the general content of the page, in `fieldContentBlock`, so it shows below this "browse by audience" section. I hardcoded the beta banner here so it will show above everything. We will have to remove it from Drupal before this work is merged.

![image](https://user-images.githubusercontent.com/1915775/103804842-b6253780-5020-11eb-9b7e-9e4117b67b32.png)